### PR TITLE
Dtspb 716  send docs death cert flow

### PR DESCRIPTION
**Pull request by meganaslattery**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPB-716

### Change description ###

Variations for 'Send Documents' page

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: app/steps/ui/documents/index.js [hunk 1]
```diff
@@ -21,29 +24,34 @@ class Documents extends ValidationStep {
...
-
-            if (!((deceasedMarried && applicantIsChild) || noDocumentsUploaded || iht205Used)) {
-                options.redirect = true;
-                options.url = '/thank-you';
-            }
+        if (!documentsRequired) {
+            options.redirect = true;
+            options.url = '/thank-you';
         }
-
```
Comment (vinceholt) [inline app/steps/ui/documents/index.js:36]: If the feature toggle is off, do the old intestacy rules not apply?

**Review by vinceholt — COMMENTED** [review]
We've got component and unit tests, do we need e2e tests?

File: app/steps/ui/documents/index.js (hunk 1 above)
Comment (meganaslattery) [reply app/steps/ui/documents/index.js:36]: I added the intestacy condition to the documents wrapper - thought it was neater in there!

Comment (meganaslattery) [reply app/steps/ui/documents/index.js:36]: https://github.com/hmcts/probate-frontend/blob/193527a354a2d9410d9bfe2f52392b6051281c13/app/wrappers/Documents.js#L18

Comment (vinceholt) [reply app/steps/ui/documents/index.js:36]: cool good move!

**Review by vinceholt — APPROVED** [review]

File: app/steps/ui/documents/template.html [hunk 2]
```diff
@@ -86,3 +161,4 @@ <h2 class="govuk-heading-m">{{ content.heading2 | safe }}</h2>
         }) }}
     </div>
 {% endblock %}
+
```
Comment (jackmunkonge) [inline app/steps/ui/documents/template.html:164]: this template definitely needs to be covered by e2e testing

**Review by jackmunkonge — APPROVED** [review]

**Pull request merged**